test/Dialect/Verif/basic.mlir

```diff
@@ -53,9 +53,69 @@ verif.formal @formal1(k = 20) {
   // CHECK: %8 = verif.concrete_input %[[C1]], %[[CLK_U]] : i1
   %8 = verif.concrete_input %c1_i1, %clk_update : i1
   // CHECK: %foo.0, %foo.1 = hw.instance "foo" @Foo("0": %6: i1, "1": %8: i1) -> ("": i1, "1": i1)
-  %foo.0, %foo.1 = hw.instance "foo" @Foo("0": %sym: i1, "1": %8 : i1)  -> ("" : i1, "1" : i1)
+  %foo.0, %foo.1 = hw.instance "foo" @Foo("0": %sym: i1, "1": %8 : i1) -> ("" : i1, "1" : i1)
 }
 
+hw.module @Bar(in %foo : i8, out "" : i8, out "1" : i8) { 
+  // CHECK: verif.contract(%foo) : (i8) {
+  verif.contract (%foo) : (i8) {
+    // CHECK: ^bb0(%[[ARG:.+]]: i8, %[[OUT:.+]]: i8, %[[OUT1:.+]]: i8):
+    ^bb0(%arg1 : i8, %bar.0 : i8, %bar.1 : i8): 
```

Comment:
Maybe i'm missing something but what do the second and third block argument mean?

Reply:
Ah ok I get it. Could you clarify that arguments correspond to output values in tablegen?